tests/tests_regression_nm.yml

```diff
@@ -36,5 +33,5 @@
     - ansible_distribution_major_version != '6'
     # The test depends on behavior that is only visible with newer NM
     - networkmanager_version is version('1.22.0', '>=')
-    - ansible_distribution != 'RedHat' or
+    - not ansible_distribution in ['RedHat', 'CentOS'] or
```

Comment:
Drop this change for now or put it into another PR ?

Reply:
Done.